Crash when creating a resource from import block

**Description**

<!-- Please add a clear and concise description of what the bug is -->

Crash when creating a resource from import block

```
╷
│ Error: Plugin did not respond
│
│ The plugin encountered an error, and failed to respond to the plugin6.(*GRPCProvider).ReadResource call. The plugin logs may contain more details.
╵

Stack trace from the terraform-provider-segment_v0.1.0 plugin:

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x8 pc=0x1024c9248]

goroutine 205 [running]:
github.com/segmentio/terraform-provider-segment/internal/provider/models.(*DestinationMetadataState).getActions(0x140001f47e0?, {0x14000170f00?, 0x6, 0x14000a97b50?})
        github.com/segmentio/terraform-provider-segment/internal/provider/models/destination_metadata.go:292 +0x178
github.com/segmentio/terraform-provider-segment/internal/provider/models.(*DestinationMetadataState).Fill(_, {{0x14000038ff0, 0x18}, {0x140005bae40, 0x1a}, {0x140005ea4e0, 0x2e}, {0x14000039008, 0x13}, {{0x140009544c0, ...}, ...}, ...})
        github.com/segmentio/terraform-provider-segment/internal/provider/models/destination_metadata.go:401 +0x170
github.com/segmentio/terraform-provider-segment/internal/provider/models.(*DestinationState).Fill(0x14000272380, 0x14000bacfe0)
        github.com/segmentio/terraform-provider-segment/internal/provider/models/destination.go:31 +0x7c
github.com/segmentio/terraform-provider-segment/internal/provider.(*destinationResource).Read(0x1400056e408, {0x102aede40, 0x14000510b40}, {{{{0x102af21b8, 0x14000593260}, {0x1029bd840, 0x140005dddd0}}, {0x102af4bb8, 0x140004ca000}}, 0x140004a4060, ...}, ...)
        github.com/segmentio/terraform-provider-segment/internal/provider/destination_resource.go:601 +0x338
github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).ReadResource(0x140002214a0, {0x102aede40, 0x14000510b40}, 0x14000510bd0, 0x14000bad5e0)
        github.com/hashicorp/terraform-plugin-framework@v1.4.0/internal/fwserver/server_readresource.go:101 +0x4a4
github.com/hashicorp/terraform-plugin-framework/internal/proto6server.(*Server).ReadResource(0x140002214a0, {0x102aede40?, 0x14000510120?}, 0x14000bdc700)
        github.com/hashicorp/terraform-plugin-framework@v1.4.0/internal/proto6server/server_readresource.go:55 +0x1a4
github.com/hashicorp/terraform-plugin-go/tfprotov6/tf6server.(*server).ReadResource(0x1400021b180, {0x102aede40?, 0x140006e3680?}, 0x140006510e0)
        github.com/hashicorp/terraform-plugin-go@v0.19.0/tfprotov6/tf6server/server.go:787 +0x3e8
github.com/hashicorp/terraform-plugin-go/tfprotov6/internal/tfplugin6._Provider_ReadResource_Handler({0x102aae720?, 0x1400021b180}, {0x102aede40, 0x140006e3680}, 0x14000472310, 0x0)
        github.com/hashicorp/terraform-plugin-go@v0.19.0/tfprotov6/internal/tfplugin6/tfplugin6_grpc.pb.go:431 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0x1400028e1e0, {0x102af3278, 0x140002a1a00}, 0x14000a10240, 0x140003f4030, 0x1030b7448, 0x0)
        google.golang.org/grpc@v1.57.0/server.go:1360 +0xc88
google.golang.org/grpc.(*Server).handleStream(0x1400028e1e0, {0x102af3278, 0x140002a1a00}, 0x14000a10240, 0x0)
        google.golang.org/grpc@v1.57.0/server.go:1737 +0x82c
google.golang.org/grpc.(*Server).serveStreams.func1.1()
        google.golang.org/grpc@v1.57.0/server.go:982 +0x84
created by google.golang.org/grpc.(*Server).serveStreams.func1
        google.golang.org/grpc@v1.57.0/server.go:980 +0x16c

Error: The terraform-provider-segment_v0.1.0 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```

**To Reproduce**

```terraform
import {
  id = "123"
  to = segment_destination.dest1
}

resource "segment_destination" "dest1" {
  name      = "dest1"
  enabled   = true
  source_id = "456abc"

  metadata = {
    id = "abc123"
  }
  settings = jsonencode({
    "api_key" : "abcd-efgh"
  })
}
```

--- a/internal/provider/models/destination_metadata.go
+++ b/internal/provider/models/destination_metadata.go
@@ -289,7 +289,7 @@ func (d *DestinationMetadataState) getActions(actions []api.DestinationMetadataA
 		destinationMetadataAction.Fields = fields
 
 		if action.DefaultTrigger.IsSet() {
-			destinationMetadataAction.DefaultTrigger = types.StringValue(*action.DefaultTrigger.Get())
+			destinationMetadataAction.DefaultTrigger = types.StringPointerValue(action.DefaultTrigger.Get())
 		}
 
 		destinationActionsToAdd = append(destinationActionsToAdd, destinationMetadataAction)

